### PR TITLE
Address TRAC-889.

### PR DESCRIPTION
**Pull request by markpbaggett**
**JIRA Ticket: [TRAC-889](https://jira.lib.utk.edu/browse/TRAC-889)

# What's it do?
This adds a new utk_mods_ir_publication_s based on genre[@authority="coar"].

# How to test

1. Modify 090  on TRACE:/scripts/custom_scripts to checkout this branch instead of institutional_repository.
2. vagrant destroy
3. vagrant up
4. Replace the MODS of an object with something that has this xml node.
5. Do a fgsupdate of the pid.
6. Query Solr for Field.

### CODE REVIEW

**Comment by CanOfBees** [comment]
@markpbaggett taking a look at this now.

File: fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt [hunk 1]
```diff
@@ -73,6 +73,13 @@
     </field>
   </xsl:template>
 
+  <!-- the following template adds a date_of_award field -->
```
Comment (CanOfBees) [inline fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt]: This is *the* _most_ ridiculous change request yet, but please change the comment :) 

`... adds a cross-collection publication type field` or something.

**Review by CanOfBees — CHANGES_REQUESTED** [review]

File: fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt (hunk 1 above)
Comment (markpbaggett) [reply fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt]: I wouldn't call that ridiculous.  If we really want these comments in there, they should represent what the template match is doing.

**Review by CanOfBees — APPROVED** [review]
@markpbaggett thanks for making this change

**Pull request merged**